Jenkinsfile-insecure.groovy

```diff
@@ -4,6 +4,10 @@ dir("packages/adminrouter/extra/src") {
     }
 
     try {
+        stage('make check-api-docs') {
```

Comment:
Yeah, we probably should ;) 

The deduplication of nginx.(master|agent).conf is going to happen soonish (<1 month) hopefully, So far almost all lua code got deduplicated.